devices/common_patches/g4_swap_IWDG_WWDG.yaml

```
@@ -0,0 +1,129 @@
+_delete:
+  - WWDG
+  - IWDG
+
+_add:
+  WWDG:
+    description: System window watchdog
+    baseAddress: 0x40002C00
+    addressBlock:
+      offset: 0x0
+      size: 0x4
```

I guess this is meant to be 1kB?

```suggestion
      size: 0x400
```

---

You're right, thanks for spotting this!

It is now corrected in the latest commit.